README.md

```diff
@@ -101,14 +132,102 @@ credentials, ie:
 }
 ```
 
-## Publishing
+The contents of this file can be found in lastpass with the name
```

Comment:
It looks exactly like what I put in there for the example.  If it gets lost the user would just have to acquire the current the current credentials and recreate the file from the example and substitute the values.  I can make that last sentence a note.